pages/pricing.tsx

```diff
@@ -63,7 +63,7 @@ const Pricing = () => {
 		return (<span className='font-money'>  {currency} <span className='text-4xl'>{price} </span ></span>);
 	}
 
-	const pricingStartDate = new Date(2023, 7, 31); // 31st August 2023
+	const pricingStartDate = new Date(2023, 9, 31); // 31st October 2023
```

Comment:
This isnt used anywhere right? its a todo?

Reply:
This is used to show the "Applicable only from" text right below the heading.